### PR TITLE
 Replace three dots by trash can in socal media feature #1409

### PR DESCRIPTION
**Pull request by dayan9265**
This pull request addresses the feature #1409: to replace the option list on the chirp with a simple delete button (with a confirmation window). This was done since currently the option list only had the delete option.

### CODE REVIEW

File: fe1-web/src/features/social/components/ChirpCard.tsx [hunk 1]
```diff
@@ -255,16 +245,23 @@ const ChirpCard = ({ chirp, isFirstItem, isLastItem }: IPropTypes) => {
...
+                    onPress={() => setShowDeleteConfirmation(true)}
                     size="small"
                     buttonStyle="secondary"
                     toolbar
                   />
+                  <ConfirmModal
+                    onConfirmPress={deleteChirp}
+                    visibility={showDeleteConfirmation}
+                    description="Are you sure you want to delete this chirp?"
+                    title="Confirm Deletion"
```
Comment (Tyratox) [inline fe1-web/src/features/social/components/ChirpCard.tsx:262]: We try to put all strings in `src/resources/strings.ts`. While this solution is not perfect it allows for easy translation of the app and without having to search the whole application for static strings :)

File: fe1-web/src/features/social/components/ChirpCard.tsx [hunk 2]
```diff
@@ -157,13 +153,7 @@ const ChirpCard = ({ chirp, isFirstItem, isLastItem }: IPropTypes) => {
 
   const listStyle = List.getListItemStyles(isFirstItem, isLastItem);
 
-  const actionSheetOptions: ActionSheetOption[] = [];
-  if (isSender && !chirp.isDeleted && isConnected) {
-    actionSheetOptions.push({
-      displayName: STRINGS.social_media_delete_chirp,
```
Comment (Tyratox) [inline fe1-web/src/features/social/components/ChirpCard.tsx:163]: Also regarding the strings file: If you delete a `STRINGS.x` statement, check if it is still used somewhere else and delete it from `strings.ts` if it is not. Maybe we can also add some linter rule to check for this in the future 🤔

**Review by Tyratox — CHANGES_REQUESTED** [review]
Great work!! 🥳 It seems you already figured out how to use react components, react hooks and even how to re-generate the snapshot tests 😁 And it is great to see that you updated the corresponding test, you even have 100% coverage of the PR 🥳😁

I just added two minor comments regarding the strings file, otherwise the PR looks good to me and I'm happy to approve it when the static strings are gone :)

For the future it would be nice to post a screenshot with a PR but it is fine for this one now 😄

**Comment by dayan9265** [comment]
Thank you for the feedback!  Indeed I did not see that we had a file with all the strings, so now normally the static strings should have been replaced and I will keep in mind that for future changes in the UI it is not a bad idea to add a screenshot to have an idea on how it looks like ^^

File: fe1-web/src/features/social/components/ChirpCard.tsx (hunk 2 above)
Comment (pierluca) [reply fe1-web/src/features/social/components/ChirpCard.tsx:163]: I like linter rules. Linters are our friends 😁 

image

**Pull request merged**